charts/refinery/templates/deployment.yaml

```diff
@@ -37,6 +37,9 @@ spec:
       {{- with .Values.podLabels }}
       {{- toYaml . | nindent 8 }}
       {{- end }}
+      {{- with .Values.redis.commonLabels}}
```

Comment:
I don't think we should be adding the redis labels to the Refinery pod

Reply:
my bad, that was a bad copy paste